tests/integration/schema_registry/test_proto_serializers.py

```diff
@@ -23,30 +23,10 @@
     PublicTestProto_pb2
 from tests.integration.schema_registry.data.proto.DependencyTestProto_pb2 import DependencyMessage
 from tests.integration.schema_registry.data.proto.exampleProtoCriteo_pb2 import ClickCas
+from tests.schema_registry.test_proto import TEST_DATA
```

Comment:
It's better to keep data in their respective test files. Its fine here to replicate data in each file if we require that.